tests/data/catalog01.ttl

```diff
@@ -0,0 +1,17 @@
+@prefix schema: <http://schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix fdp: <http://rdf.biosemantics.org/ontologies/fdp-o#> .
+@prefix dbp: <http://dbpedia.org/resource/> .
+
+<http://145.100.57.30/catalog/catalog01> a dcat:Catalog ;
```

Comment:
Are you testing against a live server? It would be better to mock the `requests` library and check that you're sending the correct request. That way it's easier to inject errors to check that they're handled appopriately, the tests can be run without an Internet connection (or when the cloud is out for maintenance), and you can check against the specification, rather than against a potentially buggy server implementation of it.

Reply:
You'r right, it's better to use mock to make the client tests isolated from the buggy server :-)